beacon-chain/state/v1/types.go

```diff
@@ -44,18 +44,28 @@ var ErrNilInnerState = errors.New("nil inner state")
 // BeaconState defines a struct containing utilities for the Ethereum Beacon Chain state, defining
 // getters and setters for its respective values and helpful functions such as HashTreeRoot().
 type BeaconState struct {
-	genesisTime           uint64
-	genesisValidatorsRoot customtypes.Byte32
-	slot                  eth2types.Slot
-	blockRoots            *customtypes.StateRoots
-	stateRoots            *customtypes.StateRoots
-	historicalRoots       customtypes.HistoricalRoots
-	eth1DepositIndex      uint64
-	balances              []uint64
-	randaoMixes           *customtypes.RandaoMixes
-	slashings             []uint64
-	justificationBits     bitfield.Bitvector4
-	state                 *ethpb.BeaconState
+	genesisTime                 uint64                      `ssz-gen:"true"`
```

Comment:
Generating SSZ for unexported fields relies on updated dependency of our protoc cast library.